install_config/aggregate_logging.adoc

```diff
@@ -1875,39 +1812,68 @@ systemctl restart docker
 
 . Update the xref:fluentd-upgrade-source[Fluentd log source].
 
-[[fluentd-upgrade-source]]
-== Updating Fluentd's Log Source After a Docker Log Driver Update
+[WARNING]
+====
+Restarting Fluentd on more than a dozen nodes at once will create a large load
+on the Kubernetes scheduler. Exercise caution when using the following the
+directions to restart Fluentd.
+====
+
+There are two methods for restarting Fluentd. You can restart the Fluentd on one
+node or a set of nodes, or on all nodes.
+
+The following steps demonstrate how to restart Fluentd on one node or a set of
+nodes.
+
+. List the nodes where Fluentd is running:
++
+----
+$ oc get nodes -l logging-infra-fluentd=true
+----
+
+. For each node, remove the label and turn off Fluentd:
++
+----
+$ oc label node $node logging-infra-fluentd-
+----
+
+. Verify Fluentd is off:
++
+----
+$ oc get ds -l component=fluentd
+----
 
-If the Docker log driver has changed from `json-file` to `journald` and Fluentd
-was previously configured with `USE_JOURNAL=False`, then it will not be able to
-pick up any new logs that are created. When the Fluentd daemonset is configured
-with the default value for `USE_JOURNAL`, then it will detect the Docker log
-driver upon pod start-up, and configure itself to pull from the appropriate source.
+. For each node, restart Fluentd:
++
+----
+$ oc label node $node logging-infra-fluentd=true
+----
+
+The following steps demonstrate how to restart the Fluentd all nodes.
 
-To update Fluentd to detect the correct source upon start-up:
+. Turn off Fluentd on all nodes:
++
+----
+$ oc label node -l logging-infra-fluentd=true --overwrite logging-infra-fluentd=false
+----
 
-. Remove the label from nodes where Fluentd is deployed:
+. Verify Fluentd is off:
 +
 ----
-$ oc label node --all logging-infra-fluentd- <1>
+$ oc get ds -l component=fluentd
 ----
-<1> This example assumes use of the default Fluentd node selector and it being
-deployed on all nodes.
 
-. Update the `daemonset/logging-fluentd` `USE_JOURNAL` value to be empty:
+. Restart Fluentd on all nodes:
 +
 ----
-$ oc patch daemonset/logging-fluentd \
-     -p '{"spec":{"template":{"spec":{"containers":[{"name":"fluentd-elasticsearch","env":[{"name": "USE_JOURNAL", "value":""}]}]}}}}'
+$ oc label node -l logging-infra-fluentd=false --overwrite logging-infra-fluentd=true
 ----
 
-. Relabel your nodes to schedule Fluentd deployments:
+. Refresh the Fluentd pod:
```

Comment:
The fluentd pods will be started automatically.   Replace this step with 'Verify Fluentd is on'
oc get pods -l component=fluentd